apps/meteor/app/apps/server/converters/rooms.js

```diff
@@ -20,6 +20,44 @@ export class AppRoomsConverter {
 		return this.convertRoom(room);
 	}
 
+	async convertRoomRaw(room) {
+		if (!room) {
+			return undefined;
+		}
+
+		const creator = room.u
+			? {
+					_id: room.u._id,
+					username: room.u.username,
+					name: room.u.name,
+				}
+			: undefined;
+
+		const type = this._convertTypeToApp(room.t);
+
+		return {
+			id: room._id,
+			slugifiedName: room.name || room.fname || room._id,
```

Comment:
This mapping for the `slugifiedName` is incorrect 🤔 Refer to the `convertRoom` method for the correct mapping